new_platforms/vsextension/ItemTemplates/oehost/host.vstemplate

```diff
@@ -0,0 +1,23 @@
+<VSTemplate Version="3.0.0" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" Type="Item">
+  <TemplateData>
+    <DefaultName>Open Enclave Host Code.c</DefaultName>
```

Comment:
Is the trailing `.c` intentional?

Reply:
yes, it was spit out by the template wizard.   If you do Add->Item... to the project it's the string filled into the dialog to save as.  I changed it to enclave_host.c.   But when you add it via by Import Enclave button, the filename is $enclavename$_host.c where $enclavename$ is the base name of the EDL you're importing.